mono/metadata/native-library.c

```diff
@@ -1249,6 +1234,11 @@ lookup_pinvoke_call_impl (MonoMethod *method, MonoLookupPInvokeStatus *status_ou
 	new_import = g_strdup (orig_import);
 #endif
 
+	if (strcmp (new_scope, "QCall") == 0) {
```

Comment:
Must this be `strcmp`?
Not an integer or enum, or a string with a length?
`strcmp` is slow.